src/renderer/reducers/queries.ts

```diff
@@ -201,30 +223,40 @@ const queryReducer: Reducer<QueryState> = function (
   }
 };
 
-function addNewQuery(state, action) {
-  if (action.reconnecting) {
-    return state;
-  }
+function addNewQuery(
+  state: QueryState,
+  queryAction: QueryAction | null,
+  connAction?: ConnectionAction,
+): QueryState {
+  let { enabledAutoComplete, enabledLiveAutoComplete, resultItemsPerPage } = INITIAL_STATE;
 
-  const configItemsPerPage = action.config && action.config.resultItemsPerPage;
```

Comment:
I couldn't find any reference for this field `resultItemsPerPage` in the Config object, so changed it to only rely on the state object for this field.

But, even though the query state object has a definition for this field, it is not really being used anywhere. We pass it down to some components, but it doesn't look like it is really being used. So, looks like we could drop off it field completely.